pkg/v1/tkg/client/upgrade_addon.go

```diff
@@ -327,14 +327,17 @@ func (c *TkgClient) setProxyConfiguration(clusterClusterClient clusterclient.Cli
 
 	if httpProxy := configmap.Data["httpProxy"]; httpProxy != "" {
```

Comment:
Does the configmap contains `TKG_HTTP_PROXY_ENABLED ` in it?

Reply:
Unfortunately, it doesn't contain this information.